tasks/systemctl_show.rb

```diff
@@ -0,0 +1,15 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+# frozen_string_literal: true
+
+require 'json'
+require 'open3'
+
+params  = JSON.parse($stdin.read)
+
+systemctl_cmd = params['bin_path']
+systemctl_args = ['show', '--no-pager', '--no-legend', '--property', params['properties'].join(','), params['service']]
```

Comment:
TIL: systemctl has a `--output` option, and you can set it to json, but it won't output json.... https://github.com/systemd/systemd/issues/83

Reply:
broken since 2015 🫣